conwayste/src/ui/mod.rs

```diff
@@ -54,22 +54,21 @@ pub enum UIAction {
     EnterText,
 }
 
-#[derive(PartialEq, Debug, Copy, Clone)]
-pub enum WidgetID {
-    MainMenuLayer1,
-    MainMenuTestButton,
-    MainMenuTestButtonLabel,
-    MainMenuTestCheckbox,
+#[derive(PartialEq, Eq, Debug, Copy, Clone)]
+pub struct WidgetID(pub usize);
 
-    MainMenuPane1,
-    MainMenuPane1ButtonYes,
-    MainMenuPane1ButtonYesLabel,
-    MainMenuPane1ButtonNo,
-    MainMenuPane1ButtonNoLabel,
+pub const MAINMENU_LAYER1: WidgetID = WidgetID(0x100);
```

Comment:
this doesn't need to be hex! :grinning:

Reply:
Also this might be better in constants.rs, because conceptually these constants are part of the API consumer, not the API itself (which is here in the `ui/` folder).

Reply:
Agreed! Will move. I personally prefer defining via hex but it's not a big deal. We'll start at '1' then.